library/ssl_tls.c

```diff
@@ -4386,7 +4401,8 @@ const char *mbedtls_ssl_get_version( const mbedtls_ssl_context *ssl )
     {
         case MBEDTLS_SSL_MINOR_VERSION_3:
             return( "TLSv1.2" );
-
```

Comment:
Maybe add "DTLSv1.3" a few lines above, too, to be forward looking for DTLSv1.3?
https://datatracker.ietf.org/doc/draft-ietf-tls-dtls13/

Reply:
I didn't add it because the library doesn't support DTLS 1.3 yet. Now that this function has a unit test, there's no risk that we forget to add it.